src/moonlink/src/table_handler/test_utils.rs

```diff
@@ -216,6 +221,24 @@ impl TestEnvironment {
         self.send_event(TableEvent::StreamAbort { xact_id }).await;
     }
 
+    /// Force an index meger, and block wait its completion.
+    pub async fn force_index_merge_and_sync(&self) {
```

Comment:
Just realized using `TableEventManager` for synchronization is much easier, will change in the followup.